docs/toc.yml

```diff
@@ -1,85 +1,48 @@
-  - name: Explore Office JS API using Script Lab
-    href: overview/explore-with-script-lab.md
-    displayName: Excel, Word, PowerPoint, Project
+  href: index.md
 
-- name: Quick Starts
+- name: Office Add-ins guidance
   items:
-  - name: Create Excel add-in
+  - name: About Office Add-ins
+    href: overview/index.md
+    displayName: Excel, Word, OneNote, Project, PowerPoint, Visio, Outlook
+  - name: Overview
     items:
-    - name: Task pane
-      href: quickstarts/excel-quickstart-jquery.md
-      displayName: Excel
-    - name: Task pane (Angular)
-      href: quickstarts/excel-quickstart-angular.md
-      displayName: Excel
-    - name: Task pane (React)
-      href: quickstarts/excel-quickstart-react.md
-      displayName: Excel
-    - name: Task pane (Vue)
-      href: quickstarts/excel-quickstart-vue.md
-      displayName: Excel
-    - name: Custom functions
-      href: quickstarts/excel-custom-functions-quickstart.md
-      displayName: Excel
-  - name: Create Word add-in
-    href: quickstarts/word-quickstart.md
-    displayName: Word
-  - name: Create OneNote add-in
-    href: quickstarts/onenote-quickstart.md
-    displayName: OneNote
-  - name: Create Outlook add-in
-    href: /outlook/add-ins/quick-start
-    displayName: Outlook
-  - name: Create PowerPoint add-in
-    href: quickstarts/powerpoint-quickstart.md
-    displayName: PowerPoint
-  - name: Create Project add-in
-    href: quickstarts/project-quickstart.md
-    displayName: Project
-
-- name: Tutorials
-  items:
-  - name: Excel add-in tutorial
-    href: tutorials/excel-tutorial.md
-    displayName: Excel
-  - name: Outlook add-in tutorial
-    href: /outlook/add-ins/addin-tutorial
-    displayName: Outlook
-  - name: PowerPoint add-in tutorial
-    href: tutorials/powerpoint-tutorial.md
-    displayName: PowerPoint
-  - name: Word add-in tutorial
-    href: tutorials/word-tutorial.md
-    displayName: Word
-
-- name: Concepts
-  items:
-  - name: Requirements for running add-ins
-    href: concepts/requirements-for-running-office-add-ins.md
-  - name: Browsers used by Office Add-ins
-    href: concepts/browsers-used-by-office-web-add-ins.md
-  - name: Add-in development best practices
-    href: concepts/add-in-development-best-practices.md
-  - name: Add-in development lifecycle
-    href: concepts/add-in-development-lifecycle.md
-  - name: Privacy and security
-    href: concepts/privacy-and-security.md
-  - name: Performance
-    href: concepts/resource-limits-and-performance-optimization.md
-
-- name: How-to guides
-  items:
+    - name: Office Add-ins platform overview
+      href: overview/office-add-ins.md
+      displayName: Excel, Word, OneNote, Project, PowerPoint, Visio, Outlook
+    - name: Host and platform availability
+      href: overview/office-add-in-availability.md
+      displayName: Excel, Word, OneNote, Project, PowerPoint, Visio, Outlook
+    - name: Explore Office JS API using Script Lab
```

Comment:
I'm inclined to wait until there's more SL content. But that said -- this will be another good A/B test to run -> Position of Script Lab node within the TOC
- A: child of "Overview" node (as-is)
- B: promoted one level up to be child of "Office Add-ins guidance" node (after 'About Office Add-ins')